packages/components/organisms/f-header/src/assets/scss/navigation.scss

```diff
@@ -166,6 +166,10 @@ $nav-link-focus-color              : $color-focus;
             text-decoration: underline;
         }
     }
+
+    @include media('<=mid') {
+        color: $nav-text-color;
+    }
```

Comment:
You could just miss out the media query for this and put it above the `@include media('>mid') {` line, so that it's effectively the default colour.